src/de/arg/value_parser.rs

```diff
@@ -0,0 +1,47 @@
+use clap::builder::ValueParser;
+use serde::Deserialize;
+
+enum_de!(ValueParser, ValueParser1,
+    #[derive(Deserialize, Clone, Copy)]
+    #[cfg_attr(feature = "kebab-case-key" ,serde(rename_all = "kebab-case"))]
+    #[cfg_attr(feature = "snake-case-key" ,serde(rename_all = "snake_case"))]
+    { }
+    {
+        Bool => {
+            ValueParser::bool()
+        },
+        String => {
+            ValueParser::string()
+        },
+        OsString => {
+            ValueParser::os_string()
+        },
+        PathBuf => {
+            ValueParser::path_buf()
+        },
+        Boolish => {
+            clap::builder::BoolishValueParser::new().into()
+        },
+        Falsey => {
+            clap::builder::FalseyValueParser::new().into()
+        },
+        NonEmptyString => {
+            clap::builder::NonEmptyStringValueParser::new().into()
+        },
+        RangedI64(s : i64 , e : i64) => {
+            (s..e).into()
+        },
+        RangedInclusizeI64(s : i64 , e : i64) => {
+            (s..=e).into()
+        },
+        RangedFromI64(s : i64) => {
+            (s..).into()
+        },
+        RangedToI64(e : i64) => {
+            (..e).into()
+        },
+        RangedToInclusizeI64(e : i64) => {
```

Comment:
What about supporting other integer types?

Reply:
I implemented the i64 because the range types of i64 was the only type that has some `From` for `ValueParser`.
Should we need for `u64` or some?

Reply:
We should support all standard numeric types.  See my other comment about limitations with `From` being a guide.

Reply:
Implemented at #43